src/patternfly/components/DataList/data-list.scss

```diff
@@ -0,0 +1,205 @@
+@import "../../patternfly-utilities";
+
+.pf-c-data-list {
+  --pf-c-data-list--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+
+  // Item
+  --pf-c-data-list__item--PaddingY: var(--pf-global--spacer--xl);
+  --pf-c-data-list__item--PaddingX: var(--pf-global--spacer--xl);
+  --pf-c-data-list__item--MarginLeft: var(--pf-global--spacer--xl);
+  --pf-c-data-list__item--BorderColor: var(--pf-global--Color--light-200);
+  --pf-c-data-list__item--BorderWidth: var(--pf-global--BorderWidth--sm);
+
+  // Cell
+  --pf-c-data-list__cell--PaddingY: var(--pf-c-data-list__item--PaddingY);
+  --pf-c-data-list__cell-cell--PaddingTop: 0;
+  --pf-c-data-list__cell-cell--md--PaddingTop: var(--pf-c-data-list__cell--PaddingY);
+  --pf-c-data-list__cell--MarginLeft: var(--pf-c-data-list__item--MarginLeft);
+
+  // Toggle
+  --pf-c-data-list__toggle--MarginRight: var(--pf-global--spacer--md);
+
+  // offset toggle to align left
+  --pf-c-data-list__toggle--MarginLeft: calc(var(--pf-global--spacer--sm) * -1);
+  --pf-c-data-list__toggle--c-button--Transition: var(--pf-global--Transition);
+  --pf-c-data-list__toggle--c-button--Transform: rotate(0deg);
+  --pf-c-data-list__toggle--c-button--m-expanded--Transform: rotate(90deg);
+
+  // Check
+  --pf-c-data-list__check--MarginRight: var(--pf-global--spacer--xl);
+
+  // Actions
+  --pf-c-data-list__action--MarginLeft: var(--pf-global--spacer--lg);
+
+  // List item border left
+  --pf-c-data-list__item--before--BackgroundColor: transparent;
+  --pf-c-data-list__item--before--Width: var(--pf-global--BorderWidth--lg);
+  --pf-c-data-list__item--before--Transition: var(--pf-global--Transition);
+  --pf-c-data-list__item--before--ZIndex: var(--pf-global--ZIndex--xl);
+
+  // Expandable content
+  --pf-c-data-list__expandable-content--PaddingY: var(--pf-global--spacer--xl);
+  --pf-c-data-list__expandable-content--MaxHeight: 0;
+  --pf-c-data-list__expandable-content--m-expanded--MaxHeight: #{pf-size-prem(600px)};
+  --pf-c-data-list__expandable-content--Transition: var(--pf-global--Transition);
+  --pf-c-data-list__expandable-content--BorderTopColor: var(--pf-global--Color--light-200);
+  --pf-c-data-list__expandable-content--BorderTopWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-data-list__expandable-content--MarginX: calc(var(--pf-c-data-list__item--PaddingX) * -1);
+
+  // Expanded modifier
+  --pf-c-data-list--m-expanded--BoxShadow: var(--pf-global--BoxShadow--md);
+  --pf-c-data-list--m-expanded--TransitionDuration: var(--pf-global--TransitionDuration);
+  --pf-c-data-list__item--m-expanded--before--BackgroundColor: var(--pf-global--active-color--100);
+
+  // spacers are a bit large on mobile view
+  // reduce spacer values for up to 380px
+  @media screen and (max-width: 380px) {
```

Comment:
Will you be reversing the order of the media queries after our discussion yesterday?